src/app/(pages)/(home)/_components/SectionVideo.tsx

```
@@ -1,66 +1,233 @@
+              });
+              setTimeout(() => {
+                toast({
+                  description: '실제 영상은 제공되지 않습니다.'
+                });
+              }, 800);
+            }}
+          >
+            <div>
+              <Image
+                src={item.imageUrl}
+                alt={item.title}
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)